src/DripsHub.sol

```diff
@@ -828,6 +829,22 @@ abstract contract DripsHub {
         return UserOrAccount({isAccount: true, user: user, account: account});
     }
 
+    function _hashUser(UserOrAccount memory userOrAccount) internal pure returns (bytes32) {
```

Comment:
This will soon be legacy and can be removed, correct?

Reply:
Correct!